### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mazedlx**
Add minor grammar improvements.

### CODE REVIEW

File: packages/pnpm/README.md [hunk 1]
```diff
@@ -2,19 +2,19 @@
...
 * **Strict.** A package can access only dependencies that are specified in its `package.json`.
 * **Deterministic.** Has a lockfile called `pnpm-lock.yaml`.
-* **Works everywhere.** Supports Windows, Linux, and macOS.
+* **Works everywhere.** It supports Windows, Linux, and macOS.
 * **Battle-tested.** Used in production by teams of [all sizes](https://pnpm.io/users) since 2016.
   
 To quote the [Rush](https://rushjs.io/) team:
 
-> Microsoft uses pnpm in Rush repos with hundreds of projects and hundreds of PRs per day, and we’ve found it to be very fast and reliable.
+> Microsoft uses pnpm in Rush repos with hundreds of projects and hundreds of PRs per day, and we've found it to be swift and reliable.
```
Comment (zkochan) [inline packages/pnpm/README.md]: It is a quote, so we cannot modify it

**Review by zkochan — COMMENTED** [review]
I will wait for a native speaker to approve this.

File: packages/pnpm/README.md [hunk 2]
```diff
@@ -2,19 +2,19 @@
 
 ![](https://i.imgur.com/qlW1eEG.png)
 
-Fast, disk space efficient package manager:
+Fast, disk space-efficient package manager:
```
Comment (illright) [inline packages/pnpm/README.md:5]: It's a complicated case as the first part consists of two words. Any chance you could provide a reference to this grammar rule?

File: packages/pnpm/README.md [hunk 3]
```diff
@@ -2,19 +2,19 @@
...
 
 * **Fast.** Up to 2x faster than the alternatives (see [benchmark](#benchmark)).
-* **Efficient.** Files inside `node_modules` are linked from a single content-addressable storage.
+* **Efficient.** Files inside `node_modules` are linked from single content-addressable storage.
```
Comment (illright) [inline packages/pnpm/README.md:8]: The article is required here, as far as I know

File: packages/pnpm/README.md [hunk 4]
```diff
@@ -2,19 +2,19 @@
...
 * **[Great for monorepos](https://pnpm.io/workspaces).**
 * **Strict.** A package can access only dependencies that are specified in its `package.json`.
 * **Deterministic.** Has a lockfile called `pnpm-lock.yaml`.
-* **Works everywhere.** Supports Windows, Linux, and macOS.
+* **Works everywhere.** It supports Windows, Linux, and macOS.
```
Comment (illright) [inline packages/pnpm/README.md:12]: This doesn't really improve grammar and makes the sentence structure inconsistent with the previous point.

File: packages/pnpm/README.md [hunk 5]
```diff
@@ -2,19 +2,19 @@
...
 * **Battle-tested.** Used in production by teams of [all sizes](https://pnpm.io/users) since 2016.
   
 To quote the [Rush](https://rushjs.io/) team:
 
-> Microsoft uses pnpm in Rush repos with hundreds of projects and hundreds of PRs per day, and we’ve found it to be very fast and reliable.
+> Microsoft uses pnpm in Rush repos with hundreds of projects and hundreds of PRs per day, and we´ve found it to be very fast and reliable.
```
Comment (illright) [inline packages/pnpm/README.md:17]: That doesn't look like an appropriate punctuation symbol for an apostrophe.

File: packages/pnpm/README.md [hunk 6]
```diff
@@ -26,15 +26,15 @@ To quote the [Rush](https://rushjs.io/) team:
 
 pnpm uses a content-addressable filesystem to store all files from all module directories on a disk.
 When using npm or Yarn, if you have 100 projects using lodash, you will have 100 copies of lodash on disk.
-With pnpm, lodash will be stored in a content-addressable storage, so:
+With pnpm, lodash will be stored in content-addressable storage, so:
```
Comment (illright) [inline packages/pnpm/README.md:29]: Again, this is questionable. Could you provide a reference?

**Review by illright — CHANGES_REQUESTED** [review]
I don't believe most of these changes are necessary and even disagree with some of them. References are definitely needed to grammar resources in order to justify such changes. Looks like a PR for the sake of a PR (for Hacktoberfest) to me :(

**Pull request closed without merging**